subprojects/docs/src/samples/java/crossCompilation/build.gradle

```diff
@@ -37,3 +40,17 @@ tasks.withType(JavaExec) {
     executable = javaExecutables.java
 }
 // END SNIPPET java-cross-compilation
+
+tasks.withType(Test) {
```

Comment:
Is this code (after `END SNIPPET`) supposed to end up in the user guide? If we want to render it then we have to move the `END SNIPPET` comment.

Reply:
Nope...It's to inject `targetJavaVersion` to the sample tests and not a part of the userguide. They are part of sample test and check.